src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -78,26 +78,25 @@ class VeteranBenefitSummaryLetter extends React.Component {
     return (
       <div>
         <p>
-          This letter shows what benefits you're receiving from the VA,
-          military service and disability status. Below, you can choose
-          if you want military service and disability stauts to be included.
+          This letter shows the benefits you're receiving from VA,
+          your military service history, and statements regarding your disability status. You can choose
+          what information you want to include in your letter.
         </p>
         <h2>Choose the information you want to include.</h2>
+        <h2>Military Service Information</h2>
         <p>
-          Our system shows the most recent periods of service. There may be later
-          periods of service that aren't displayed here.
-        </p>
-        <p>
-          <strong>Please note:</strong>This letter can only show up to the 3 most
-          recent periods of service.
+          The 3 most recent periods of service are available to show in your letter. Select whether or not you want them included.
         </p>
-        <h2>Military Service Information</h2>
+
         {militaryServiceInfo}
         <h2>VA Benefit Information</h2>
+        <p>
+          Select which statements you want to include in your letter.
+        </p>
         {vaBenefitInformation}
         <p>
           If you see incorrect information for service periods or disability status,
-          please send a question using VA's <a href="/">Inquiry Routing & Information
+          please send a question using VA's <a href="https://iris.custhelp.com/app/ask">Inquiry Routing & Information
```

Comment:
Can we please also add the property `target="_blank"` to this `<a>`? For an example of what this should look like, see above for the address update link.